common/interfaces.go

```diff
@@ -10,9 +10,15 @@ import (
 	"github.com/roadrunner-server/sdk/v4/state/process"
 	"github.com/roadrunner-server/sdk/v4/worker"
 	"github.com/temporalio/roadrunner-temporal/v4/internal"
+	"go.temporal.io/sdk/interceptor"
 	"go.uber.org/zap"
 )
 
+type TemporalInterceptor interface {
+	TemporalInterceptor() interceptor.Interceptor
+	Name() string
+}
```

Comment:
Since this is the worker interceptor (temporal worker), and method name like you did in the `grpc`

```go
type WorkerInterceptor interface {
    Interceptor() ...
    Name() ...
}
```

Reply:
Since the same name exists for gRPC I am not able to implement new method in otel plugin.

Reply:
Ah, I forgot that we're not in Rust... I'll need to think about how to name all these new interceptors. Personally, I don't like naming it as `TemporalInterceptor`. Might be we may find a better name...

Reply:
Ok, let's call it `Interceptor`, and method -> `WorkerInterceptor`. I also updated the `grpc` and `otel` parts to fit the current naming.
```go
type Interceptor interface {
    WorkerInterceptor() ...
    Name() ...
}
```

Reply:
Done